src/seedu/addressbook/AddressBook.java

```diff
@@ -721,7 +731,7 @@ private static void createFileIfMissing(String filePath) {
     private static ArrayList<String> getLinesInFile(String filePath) {
         ArrayList<String> lines = null;
         try {
-            lines = new ArrayList(Files.readAllLines(Paths.get(filePath)));
+            lines = new ArrayList<String>(Files.readAllLines(Paths.get(filePath)));
```

Comment:
Java does give some warnings for not specifying the data type. So it is a valid change here